### ISSUE TITLE
Eigengrau's Helper is currently not working due to broken library changes.

### ISSUE THREAD

**Issue by ryceg** [ryceg/Eigengrau-s-Essential-Establishment-Generator#669]
**Describe the bug**

[Eigengrau's Helper](https://github.com/ryceg/Eigengraus-Helper) is a companion Discord bot which is currently not working due to libraries having breaking changes. It needs to be fixed so that the Hivemind Creativity channels can work again. 

I commissioned the bot from someone else, so have only a very passing understanding of how the Discord API works, but my understanding is that a third party API was used to enable slash commands (because it was commissioned when slash commands were still in beta); the issue therefore likely lies in the `slash-create` module. 

The first to submit a pull request that fixes the issues and restores it to previous functionality will receive the bounty of $50USD payable via PayPal or similar method.

**Comment by Tasups** [comment]
Is this issue still open/need to be fixed?

**Comment by ryceg** [comment]
Yes, it is indeed still open. If you want to take a crack at it, shoot me a message over in the Discord server and I can set up some channels for you to use to test.
